script/start-sample

```diff
@@ -34,7 +34,7 @@ set -e
 
 sampledir=examples/sample
 
-while getopts :bo opt
+while getopts :boe opt
```

Comment:
prefer adding a comment explaining what `:boe` is..

Reply:
uhm, this is your script. I can add a reference to the bash man pages. Maybe you meant to add a comment in the usage as to what the -e option is.

Reply:
Ah.. right.. yes..

Reply:
(the reference to bash man pages is not required..)